tasks/support/leaseweb.rb

```diff
@@ -42,6 +42,15 @@ def leaseweb_init
              retry
            end
 
+           hardware_details = nil
+
+           begin
+             hardware_details = account_api.getV2DedicatedServerHardware(info['id']) until hardware_details && !hardware_details['errorCode']
```

Comment:
This does not seem to be working (`undefined method 'getV2DedicatedServerHardware'`)...

We need to query [this api endpoint](http://developer.leaseweb.com/api-docs/baremetal_v2.html#get-servers12345hardwareInfo) for hw info, but our [leaseweb lib](https://github.com/liquidm/leaseweb-rest-api/blob/master/lib/leaseweb-rest-api.rb) does not support it yet.